create_infotexts.py

```diff
@@ -371,7 +371,11 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
         place_as_wikitext = ""
 
         if place_string:
-            if place_string in places_mapping:
+            # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
+            if places_mapping[place_string].get('commonscat'):
+                self.content_cats.append(places_mapping[place_string]["commonscat"])
+
+            if place_string.lower() in [place.lower() for place in places_mapping.keys()]:
```

Comment:
Is there really a mixture of upper/lower in both the mapping and the in-data?

If so I would standardise the mapping data (just make them all lower). Then in the `if place_string in places_mapping` above (see previous comment) simply change `place_string` to `place_string.lower()` (and drop this if).